hieradata/cluster/chango/role/rke.yaml

```diff
@@ -0,0 +1,6 @@
+---
+classes:
+  - "profile::core::sysctl::rp_filter"
+profile::core::sysctl::rp_filter::enable: false
+profile::core::rke::enable_dhcp: true
+profile::core::rke::version: "1.4.6-rc4"
```

Comment:
This single line can be moved to `cluster/chango.yaml` and then this file could be removed completely.

Reply:
`profile::core::rke::version: "1.4.6-rc4"` was moved to cluster/chango.yaml